python/maps_creator_atlas_data.py

```diff
@@ -0,0 +1,120 @@
+"""
+Update content of the Atlas - generate maps based on the preprocessed netcdf data
+Function        : Plot maps in a uniform way based on the netcdf files
+Author          : Team BETA
+First Built     : 2021.08.12
+Last Update     : 2021.08.13
+Library         : os, glob, netcdf4, matplotlib, cartopy, argparse
+Description     : In this notebook serves to extract netcdf data and generate maps
+                  for Atlas page.
+Return Values   : png files
+Note            : All the maps are generated in a uniform way.
+"""
+
+import os
+import argparse
+import glob
+from netCDF4 import Dataset
+import matplotlib
+import matplotlib.pyplot as plt
+import cartopy.crs as ccrs
+# Generate images without having a window appear
+matplotlib.use('Agg')
+
+def files(datapath, output_path):
+    """
+    Generate maps in a uniform way based on the netcdf files.
+    """
+    # get the list of .nc files in the target directory
+    #nc_files = glob.glob(os.path.join(datapath,"*.nc"))
+    all_files = os.listdir(datapath)
```

Comment:
Thanks for the comment! I agree that it is always nice to use `pathlib` in python 3, as a replacement of the low-level path manipulation tool `os.path`. I have updated the script using `pathlib.Path` instead of `os.path`.